RallySimulator.Application/Core/Vehicles/Queries/GetVehicles/GetVehiclesQuery.cs

```diff
@@ -151,8 +151,8 @@ private static string ValidateOrderBy(string orderBy)
                 }
 
                 string sortDirection = orderByPart.EndsWith(" desc", StringComparison.OrdinalIgnoreCase) ? " desc" : string.Empty;
-
-                orderByStringBuilder.Append($"{ValidOrderByColumnsDictionary[columnName]}{sortDirection}, ");
+                string orderByValue = $"{ValidOrderByColumnsDictionary[columnName]}{sortDirection}, ";
```

Comment:
Do you think we can do `ValidOrderByColumnsDictionary.TryGetValue` instead of `ValidOrderByColumnsDictionary.ContainsKey` just above?

That way we can access the value captured as the `out` parameter instead of having two dictionary lookups.